invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/PatronCurrentLoans/state/actions.js

```diff
@@ -17,6 +14,7 @@ export const fetchPatronCurrentLoans = patronPid => {
           .qs()
       )
       .then(response => {
+        console.log('GOT RESPONSE');
```

Comment:
I think this is to be removed... even if it should be stripped out by webpack...